Dockerfile

```diff
@@ -1,13 +1,12 @@
 FROM eclipse-temurin:11 AS builder
 
-RUN apt-get -y update && apt-get install unzip
-
+RUN apt-get -y update && apt-get install unzip 
 WORKDIR /app
 
 # install ballerina
-RUN wget --no-verbose https://dist.ballerina.io/downloads/swan-lake-beta3/ballerina-swan-lake-beta3.zip
-RUN unzip ballerina-swan-lake-beta3.zip
-ENV PATH="${PATH}:/app/ballerina-swan-lake-beta3/bin"
+RUN wget --no-verbose https://dist.ballerina.io/downloads/2201.2.0/ballerina-2201.2.0-swan-lake.zip
```

Comment:
should we refactor these commands to read the version number from an env var? (could be supplied with ARG to only affect build time)